src/legacy/core_plugins/timelion/public/plugin.ts

```diff
@@ -69,8 +68,8 @@ export class TimelionPlugin implements Plugin<Promise<void>, void> {
     const dependencies: TimelionVisualizationDependencies = {
       uiSettings: core.uiSettings,
       http: core.http,
-      timefilter: data.timefilter,
       timelionPanels,
+      data,
```

Comment:
Curious, why you decided to use here `data` plugin instead of just `timefilter` service as before. Other entries in this interface (`uiSettings`, `http`) are service, instead of whole `core` being passed in.

Reply:
fixed